ztp/resource-generator/Containerfile

```diff
@@ -0,0 +1,32 @@
+FROM quay.io/openshift/origin-cli:latest
+
+USER root
+
+# For CLI use in pre-sync
+RUN INSTALL_PKGS="tar unzip golang jq" && \
+    yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
+    rpm -V $INSTALL_PKGS && \
+    yum clean all -y && \
+    curl -sSLo - https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/v4.1.3/kustomize_v4.1.3_linux_amd64.tar.gz | \
+        tar xzf - -C /usr/local/bin
+    
+# Create working directory
+RUN mkdir /usr/src/hook
+
+WORKDIR /usr/src/hook
+# For Python client
+RUN python -m venv venv && \
+    source venv/bin/activate && \
```

Comment:
Since this is in a container is a virtualenv needed or can kubernetes and jinja2 be installed directly in the container OS?

Reply:
https://issues.redhat.com/browse/CNF-2728